iceoryx_utils/include/iceoryx_utils/cxx/forward_list.hpp

```diff
@@ -0,0 +1,402 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef CXX_FORWARD_LIST_HPP_INCLUDED
+#define CXX_FORWARD_LIST_HPP_INCLUDED
```

Comment:
Include guard should look like this "IOX_UTILS_CXX_FORWARD_LIST_HPP"

Reply:
done in 80f54bf97462be9eb59d71c1dfabdc2beab54efd